multiapps-mta/src/main/java/org/cloudfoundry/multiapps/mta/model/Version.java

```diff
@@ -32,21 +27,12 @@ public int getPatch() {
         return version.getPatch();
     }
 
-    public String getBuild() {
-        return version.getBuild();
-    }
-
-    public String[] getSuffixTokens() {
-        return version.getSuffixTokens();
-    }
-
     public static Version parseVersion(String versionString) {
-        try {
-            String fullVersionString = PARTIAL_VERSION_CONVERTER.convertToFullVersionString(versionString);
-            return new Version(new Semver(fullVersionString, SemverType.NPM));
-        } catch (SemverException e) {
-            throw new ParsingException(e, Messages.UNABLE_TO_PARSE_VERSION, versionString);
+        var version = Semver.coerce(versionString); //allows incomplete version strings like "3" or "3.0"
```

Comment:
Will this work with some strange version like 3.1.0+123123, because it might be breaking